test/Form_test.js

```diff
@@ -229,6 +229,27 @@ describe("Form", () => {
       expect(node.querySelectorAll("input[type=text]")).to.have.length.of(1);
     });
 
+    it("should handle references to deep schema definitions", () => {
+      const schema = {
+        definitions: {
+          testdef: {
+            type: "object",
+            properties: {
+              bar: { type: "string" },
+            },
+          },
+        },
+        type: "object",
+        properties: {
+          foo: { $ref: "#/definitions/testdef/properties/bar" },
```

Comment:
I couldn't find anything in the spec either, but I tried the given example on https://jsonschemalint.com/ and it works the way you would expect it to.

[Section 8.2.1](http://json-schema.org/latest/json-schema-core.html#rfc.section.8.2.1) says "Schemas can be identified by any URI that has been given to them, including a [JSON Pointer](https://tools.ietf.org/html/rfc6901)".